harness/features/variable.local.test.ts

```diff
@@ -0,0 +1,356 @@
+import { 
+    getConnectionStringForProxy, 
+    forEachSDK, 
+    describeIf, 
+    createClient, 
+    createUser, 
+    callVariable, 
+    callOnClientInitialized,
+    forEachVariableType,
+    waitForRequest
+} from '../helpers'
+import { Capabilities, SDKCapabilities } from '../types'
+import { v4 as uuidv4 } from 'uuid'
+import { getServerScope } from '../nock'
+import { config } from '../mockData/index'
+import nock from 'nock'
+
+jest.setTimeout(10000)
+
+const scope = getServerScope()
+
+const expectedVariablesByType = {
```

Comment:
maybe use the `BucketedConfig['variables']` type here like I added in the mock data

Reply:
oh nvm I see this is not that type